README.md

```diff
@@ -21,11 +29,16 @@ Navigate to the **Config** tab of your robot’s page in [the Viam app](https://
 > [!NOTE]  
 > For more information, see [Configure a Robot](https://docs.viam.com/manage/configuration/).
 
+To save your changes, click **Save config** at the bottom of the page.
+Check the **Logs** tab of your robot in the Viam app to make sure your RPlidar has connected and no errors are being raised.
+
 ### Attributes
 
 No attributes are available for a `lidar:rplidar` camera.
 
-## Local installation
+## Build & Run Locally
```

Comment:
[opt] To match the `Build and Run` section above, we might want to consider changing this:
```suggestion
## Build and Run locally
```

or
```suggestion
## Build and Run Locally
```

Though we have a `Build & run rplidar as a modular component` subsection below, I don't feel strongly about this. (Or we also change that to `Build and run rplidar as a modular component`. Up to you!)